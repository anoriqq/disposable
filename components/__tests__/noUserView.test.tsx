import React from 'react';
import renderer from 'react-test-renderer';

import NoUserView from '../noUserView';

describe('UserView', () => {
  it('Snapshot', () => {
    const markdownBody = '# README';
    const tree = renderer
      .create(<NoUserView markdownBody={markdownBody} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
