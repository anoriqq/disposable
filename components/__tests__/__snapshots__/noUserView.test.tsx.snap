// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserView Snapshot 1`] = `
Array [
  <a
    href="/user/auth"
  >
    <img
      alt="login"
      src="/btn_google_signin_light_normal_web.png"
    />
  </a>,
  <div
    style={
      Object {
        "margin": "0 10px",
      }
    }
  >
    <span>
      By clicking â€œSign in", you agree to our 
      <a
        href="/terms"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Terms of Service
      </a>
       and 
      <a
        href="/privacy"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Privacy Policy
      </a>
      .
    </span>
  </div>,
  <div>
    <h1>
      README
    </h1>
  </div>,
]
`;
